BepInEx.Core/Contract/PluginInfo.cs

```diff
@@ -33,24 +38,23 @@ public class PluginInfo : ICacheable
     /// </summary>
     public IEnumerable<BepInIncompatibility> Incompatibilities { get; internal set; }
 
-    /// <summary>
-    ///     File path to the plugin DLL
-    /// </summary>
-    public string Location { get; internal set; }
-
+    public string TypeName { get; internal set; }
+    
     /// <summary>
     ///     Instance of the plugin that represents this info. NULL if no plugin is instantiated from info (yet)
     /// </summary>
     public object Instance { get; internal set; }
-
-    public string TypeName { get; internal set; }
-
+    
     internal Version TargettedBepInExVersion { get; set; }
+    
+    internal string Location { get; set; }
+    
+    /// <inheritdoc />
+    public override string ToString() => $"{Metadata?.Name} {Metadata?.Version}";
 
-    void ICacheable.Save(BinaryWriter bw)
+    public virtual void Save(BinaryWriter bw)
```

Comment:
Why change the caching methods to public? They shouldn't have any use outside of BepInEx.

Reply:
I think this is a mistake because I previously messed a lot with the caching system, but those changes ended up not being necessary. Will recheck and revert if it is